webpack/assets/javascripts/react_app/redux/reducers/index.js

```diff
@@ -4,9 +4,14 @@ import hosts from './hosts';
 import notifications from './notifications/';
 import toasts from './toasts';
 
-export default combineReducers({
-  statistics,
-  hosts,
-  notifications,
-  toasts,
-});
+export function combineReducersAsync(asyncReducers) {
+  return combineReducers({
+    statistics,
+    hosts,
+    notifications,
+    toasts,
+    ...asyncReducers,
+  });
+}
+
+export default combineReducersAsync();
```

Comment:
Does it work? Shouldn't it be:
```js
export default combineReducersAsync({});
# or
export function combineReducersAsync(asyncReducers = {}) { ... }
```

Reply:
Yup - it is a little surprising but the spread operator doesn't mind if the value it's spreading is undefined. It will just skip it.